service/awsconfig/v10/ebs/error.go

```diff
@@ -12,8 +12,16 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+var volumeNotFoundError = microerror.New("volume not found")
+
 // IsVolumeNotFound asserts volume not found error from upstream's API code.
 func IsVolumeNotFound(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	c := microerror.Cause(err)
+
 	aerr, ok := err.(awserr.Error)
```

Comment:
Lets assert against the cause here when we already have it.

Reply:
Done.